views/layout.html

```diff
@@ -25,7 +25,7 @@
   <link href="{{assetPath}}/stylesheets/govuk-frontend/v3.11.0/govuk-frontend-ie8-3.11.0.min.css" rel="stylesheet" />
   <![endif]-->
 
-  <script src="https://code.jquery.com/jquery-3.4.1.js"
+  <script src="https://code.jquery.com/jquery-3.6.0.js"
           integrity="sha256-WpOohJOqMqqyKL9FccASB9O0KwACQJpFTUBLTYOVvVU="
```

Comment:
have you checked that the browser actually loads in jquery here?
The integrity hash should have changed between versions. So I expect the browser would block this from running because 3.6.0's sha will be different from this

Reply:
![Screen Shot 2021-06-28 at 08 41 27](https://user-images.githubusercontent.com/38655987/124889461-06c91d80-dfcf-11eb-80bc-4f112252661b.png)

Reply:
![Screen Shot 2021-07-08 at 09 28 57](https://user-images.githubusercontent.com/38655987/124889520-134d7600-dfcf-11eb-8b9c-42ed26220cf1.png)

Reply:
Where do we get the keys from?

Reply:
Updated the hash